### DISCUSSION TITLE
Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https://bronyfurry.com".

### DISCUSSION THREAD

**Discussion by DarkShyMW** [mastodon/mastodon#18454] [Q&A]
Hello!
I was install mastodon on clear debian server.
When I logged into my account for the first time - everything was fine, after that I decided to check from my phone - a white page opened without any content.
After that, I logged out of my PC account and saw the same white screen.
In console:
`Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https://bronyfurry.com". Either the 'unsafe-inline' keyword, a hash ('sha256-VgAdUhL1FabDOMlyP5PHEqy0KxxjG1nQBuqXlYPNwVs='), or a nonce ('nonce-...') is required to enable inline execution.`
`Failed to find a valid digest in the 'integrity' attribute for resource 'https://bronyfurry.com/packs/js/application-d735d94afb9092eeeb5e.chunk.js' with computed SHA-256 integrity 'e6746tuillKCb87cCcrrqOLAsJT9edD9RLTxF3HnQgg='. The resource has been blocked.
`
and same errors. When i clear cache, sites works 1-2 minutes and down again.
you can try to watch it:
https://bronyfurry.com

**Comment by kurucu** [comment]
It seems you fixes this - what did you do?
